stable/filebeat/values.yaml

```diff
@@ -1,24 +1,13 @@
 image:
   repository: docker.elastic.co/beats/filebeat-oss
-  tag: 6.7.0
+  tag: 7.0.1
   pullPolicy: IfNotPresent
 
 config:
-  filebeat.config:
```

Comment:
has this been removed in filebeat 7.0.1?

Reply:
Yes but let me confirm that tomorrow. 

I'll do some more checks.

Reply:
You were right, only filebeat.config.prospectors has been removed. 

Thanks !